src/jit/codegencommon.cpp

```diff
@@ -2465,6 +2465,21 @@ bool CodeGen::genCreateAddrMode(GenTreePtr  addr,
         return false;
     }
 
+#if defined(_TARGET_ARM_) && !defined(LEGACY_BACKEND)
+    if (mode == -1)
+    {
```

Comment:
Because it called twice for the same instuction (first time with `mode == 0` on early stages).
I will provide more details tomorrow or you can remove this `if (mode == -1)` and put break point to return statement.